rllib/algorithms/algorithm_config.py

```diff
@@ -344,6 +344,8 @@ def __init__(self, algo_class=None):
         self.enable_connectors = True
         self._env_to_module_connector = None
         self._module_to_env_connector = None
+        self.add_default_connectors_to_env_to_module_pipeline = True
```

Comment:
3 new config options to completely disable default connector pieces to be added by RLlib. This is for advanced users who know exactly what they are doing (and why they don't need these pieces).